Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -197,7 +197,7 @@ fileprivate enum TaskMetadata: DependencyTracker {
     case let notification as DidSaveTextDocumentNotification:
       self = .documentUpdate(notification.textDocument.uri)
     default:
-      logger.error(
+      logger.log(
```

Comment:
For cases we're specifically ignoring, would it be worth having a case here and then keeping the error for truly unknown cases?

Reply:
Yeah, might be the better option. I added cases for all known notifications and requests now.